src/System.Net.Sockets/src/RefactoringShims/InteropShim.cs

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
VS didn't show any reference... it's probably caused by the bug that switched "/" with "\" within dir.targets.
I tried removing it and now I understand what's going on: you are still including some of the classes I mistakenly placed in Interop\Winsock. 

I think we'll need to PAL these as well and move them to common.